apps/src/lab2/projects/ProjectContainer.tsx

```diff
@@ -70,6 +88,35 @@ const ProjectContainer: React.FunctionComponent<ProjectContainerProps> = ({
     });
   }, []);
 
+  useEffect(() => {
+    // Ensure the header is cleared when we have a change,
+    // then possibly load a new header if the level has one.
+    dispatch(clearHeader());
+    // If there is no channel, we can't load a header.
+    // TODO: Handle the case of viewing another user's project.
+    // Currently we aren't loading level data for those projects, so we don't
```

Comment:
I thought the only time we didn't load level data was on pages like /projectbeats? Don't we still load level data on standalone pages even if the user isn't the project owner?

Reply:
I'll update this comment, we don't seem to load level data for the share url (which does not have `/view` appended), but we do for the `/view` url.